README.md

```diff
@@ -48,11 +48,12 @@ jobs:
        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
 ```
-In case you are using Maven or Gradle scanner in your repository, you should alter the location of the report metadata file by using the optional `scanMetadataReportFile` input.
+In case you are using other scanner than [SonarScanner CLI](https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/) in your repository, you should alter the location of the report metadata file by using the optional `scanMetadataReportFile` input.
```

Comment:
The user doesn't now that the action uses SonarScanner CLI. Wouldn't it be better to just say "When using this action with `sonarsource/sonarqube-scan` action or with C/C++ code analysis you don't have to provide `scanMetadataReportFile`, otherwise do this and that..."?

Reply:
Yeah, make sense, I'll try to rephrase it